frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/TaskList.tsx

```diff
@@ -74,7 +69,9 @@ const TaskList: React.FC<any> = ({
         data-test="task-list"
         ref={hoverRef}
         className="odc-task-list-node__trigger"
-        onClick={() => setMenuOpen(!isMenuOpen)}
+        onClick={() => {
+          setTimeout(() => onTaskSearch(onNewTask), 0); // wait for thread end
```

Comment:
Hmmm, why do we need to worry about thread end? It's gotta be a shallow thread from an onClick of a 'g' tag.

Reply:
Right, we don't need to wait on this. fixed the issue.